content/docs/start/experiments/experiment-tracking.md

```diff
@@ -99,7 +123,9 @@ containing the results and the changes needed to reproduce it.
 Framework and any
 [data tracked by DVC](/doc/start/data-management/data-versioning) but you can
 also [log additional info](/doc/dvclive#log-data) to be included in the
-experiment.
+experiment. `live.log_artifact("mymodel", type="model")` will
+[track your model with DVC](/doc/dvclive#log-data) and enable managing it with
```

Comment:
```suggestion
[track your model with DVC](/doc/dvclive/live/log-artifact) and enable managing it with
```

The current link just goes to a similar code snippet that then refers to /doc/dvclive/live/log-artifact.